util/hash.go

```diff
@@ -14,6 +17,20 @@ type RLPHashable interface {
 	RLPHash() Hash
 }
 
+// Custom JSON unmarshal needed for test fixtures
+func (h *Hash) UnmarshalJSON(data []byte) error {
+	s := string(data[1:len(data)-1])
```

Comment:
I'm assuming that this goes from a `0x` hex string to bytes?

Reply:
Not exactly, the json parser sends the entire field (string quotes included.)